app/service/rest_svc.py

```diff
@@ -134,9 +134,10 @@ def __init__(self):
         return dict(contacts=self.get_service('contact_svc').report.get(contact.get('contact'), dict()))
 
     async def update_agent_data(self, data):
-        if 'paw' not in data:
+        paw = data.pop('paw', None)
```

Comment:
move this small change to a PR

Reply:
https://github.com/mitre/caldera/pull/1427